swarm/api/routes/db.py

```
@@ -297,6 +304,10 @@ async def ingest_run_events(run_id: str):
             "timestamp": datetime.now(timezone.utc).isoformat(),
         }
 
+    except ValueError as e:
+        raise HTTPException(status_code=400, detail=str(e))
```

Ruff (0.14.14)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)